test/jasmine/assets/mouse_event.js

```diff
@@ -1,11 +1,16 @@
-module.exports = function(type, x, y) {
-    var options = {
+module.exports = function(type, x, y, opts) {
+    var fullOpts = {
         bubbles: true,
         clientX: x,
         clientY: y
     };
 
+    // https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent
+    if(opts) {
+        if(opts.buttons) fullOpts.buttons = opts.buttons;
```

Comment:
https://developer.mozilla.org/en-US/docs/Web/API/MouseEvent/buttons

This is what the gl-vis relies on to determine which mouse over is considered a click event.